### ISSUE TITLE
lodash has replaced _contains to _includes

### ISSUE THREAD

**Issue by chauhan216** [mgonto/restangular#1307]
lodash has replace _contains to _includes in its latest version. This is being used in restangular file so throwing error with undefined function.
        return _.contains(safeMethods, operation.toLowerCase());

**Comment by fabienvauchelles** [comment]
+1

**Comment by diegopamio** [comment]
+1

**Comment by cayblood** [comment]
+1

**Comment by Blackbaud-RobertTester** [comment]
+1

**Comment by jeton** [comment]
+1

**Comment by Uelb** [comment]
+1

**Comment by shepilov-vladislav** [comment]
:+1:

**Comment by MRHarrison** [comment]
+1

**Comment by Climax777** [comment]
Same problem here +1

**Comment by fintis** [comment]
+1

**Comment by alimek** [comment]
+1

**Comment by pelim** [comment]
+1

**Comment by pelim** [comment]
why you are not fixing the loaddash version to one implementing ._contains

**Comment by diegopamio** [comment]
Because I want to use 4.x version features.

**Comment by spywen** [comment]
+1

**Comment by aroucadi** [comment]
+1

**Comment by sushantdhiman** [comment]
Fix for now in `app.config` phase

 ```
//until https://github.com/mgonto/restangular/issues/1307
 _.contains = _.includes;
```

**Comment by bsonntag** [comment]
+1

**Comment by andyk1989** [comment]
+1

**Comment by adko-pl** [comment]
+1

**Comment by ConcurrentHashMap** [comment]
Will be fixed by merging #1314.

**Comment by ConcurrentHashMap** [comment]
Good news @chauhan216 and all the others following this thread: The issue is finally fixed. I released v1.5.2 that addresses this issue. I will close this thread for now; feel free to reopen a new issue if something doesn't work as expected.

Thanks for your help and patience!
